### DISCUSSION TITLE
Caddy - Self Signed Cert - REST Self-consumption, unable to get issuer certificate

### DISCUSSION THREAD

**Discussion by Shadowfita** [Budibase/budibase#12542] [Help] [answered]
Hi,

Just wondering if there is an easy way to install self-signed certs onto budibase for consuming the budibase REST API in itself?

I've tried 
      - NODE_EXTRA_CA_CERTS=/app/cert.crt

Under app-service, worker-service and proxy-service, but no change.

Thank you.

**Comment by melohagan** [comment, marked as answer]
You could explore building a custom datasource? https://docs.budibase.com/docs/custom-datasource
